rskj-core/src/main/java/org/ethereum/facade/EthereumFactory.java

```diff
@@ -19,7 +19,6 @@
 
```

Comment:
This Class and `org.ethereum.Start` can be safely removed, they were part of how _EthereumJ_ was intialized, and also it's dangerous to have another class than `co.rsk.Start` creating an Application Context